processor/processor.go

```diff
@@ -1473,6 +1478,16 @@ type storeMessage struct {
 }
 
 func (proc *HandleT) Store(in storeMessage) {
+	ctx, cancel := context.WithTimeout(context.TODO(), proc.storeTimeout)
+	defer cancel()
+
+	go func() {
+		<-ctx.Done()
+		if ctx.Err() == context.DeadlineExceeded {
+			panic(fmt.Sprintf("processor .Store() timed out after %s", proc.storeTimeout))
```

Comment:
As we store data into multiple tables (multiple transactions?), we might be writing some duplicates, depending on which query caused the panic.

Reply:
That is true, but I believe this is expected, to happen in abnormal situations. Otherwise, we need to introduce transactions.

Since we have seen this happen on batchrouter as far as I know, we can switch the order to avoid duplicates in this case.

Reply:
I have switched the order of inserts, this would make double writes less likely